### ISSUE TITLE
Could you provide an example with TSL or SSL?

### ISSUE THREAD

**Issue by diuming** [IBM-Swift/Swift-SMTP#96]
I'm testing Swift-SMTP with Gmail in two situations. (less secure app access is `on` and `off`)

1)less secure app access is `on`
   The Swift-SMTP works very well
2)less secure app access is `off`
    The Swift-SMTP will cause an issue, please refer to following

So, could you provide an example for situation 2

issue with gmail (when less secure app access is off)
---
Bad response received for command. command: (ZXdvdGRnYWJEVw==), response: 534-5.7.9 Application-specific password required.

**Comment by quanvo87** [comment]
I don't think it is possible for it to work with less secure app access turned off.
